File: FrontEnd/src/app/DTO/editUser.dto.ts
export class EditUserDTO {
  id: number;
  email: string;
  firstName: string;
  lastName: string;
  dateOfBirth: string;
  phoneNumber: number;
  createdDateTime?: Date;

  constructor(data: any) {
    this.id = data.id || 0;
    this.email = data.email || '';
    this.firstName = data.firstName || '';
    this.lastName = data.lastName || '';
    this.dateOfBirth = data.dateOfBirth || '';
    this.phoneNumber = data.phoneNumber || 0;
    this.createdDateTime = data.createdDateTime ? new Date(data.createdDateTime) : undefined;
  }
}

File: BackEnd/DTO/EditUserDTO.cs
namespace SkydiveEquipmentStore.DTO
{
    public class EditUserDTO
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string DateOfBirth { get; set; }
        public int PhoneNumber { get; set; }
        public DateTime? CreatedDateTime { get; set; } = DateTime.Now;
    }
}
